src/app/CommandSender.cpp

```diff
@@ -32,9 +32,10 @@
 namespace chip {
 namespace app {
 
-CommandSender::CommandSender(Callback * apCallback, Messaging::ExchangeManager * apExchangeMgr, bool aIsTimedRequest) :
-    mExchangeCtx(*this), mpCallback(apCallback), mpExchangeMgr(apExchangeMgr), mSuppressResponse(false),
-    mTimedRequest(aIsTimedRequest)
+CommandSender::CommandSender(Callback * apCallback, Messaging::ExchangeManager * apExchangeMgr, bool aIsTimedRequest,
+                             bool suppressResponse) :
```

Comment:
This is not supported on the server, right?  See https://github.com/project-chip/connectedhomeip/issues/8043

And presumably we should call this aSuppressResponse and remove it from AddRequestDataNoTimedCheck?

Reply:
And remove from AddRequestDataNoTimedCheck and change the caller of that to pass it to the constructor instead?

Reply:
Yup - this is now removed.